### ISSUE TITLE
rocket chat bot wakes me up in the middle of the night

### ISSUE THREAD

**Issue by gstlouisgit** [RocketChat/Rocket.Chat#18969]
### Description:

This morning at 3AM rocket chat on my phone woke me up because there is a new release. There is ALWAYS a new release. How can we disabled this please? Or at least respect the timezones as this is getting ridiculous

### Steps to reproduce:

android app

### Expected behavior:

at least the ability to disable but I can't find it 

### Actual behavior:

WAKES ME UP IN THE MIDDLE OF THE NIGHT

### Server Setup Information:

- Version of Rocket.Chat Server:  3.5.*
- Operating System: centos
- Deployment Method: manual install
- Number of Running Instances: 1
- DB Replicaset Oplog: ?
- NodeJS Version: latest
- MongoDB Version: latest

### Client Setup Information

- Desktop App or Browser Version: latest
- Operating System: android

**Comment by luixxiul** [comment]
I would rather enable Do Not Disturb mode with automatic rules not to let apps send notifications midnight, as any of them could wake you up.

**Comment by ankar84** [comment]
> I would rather enable Do Not Disturb mode with automatic rules not to let apps send notifications midnight, as any of them could wake you up.

Indeed. Agree that it's not Rocket chat problem itself. Anyone can write in any messanger. So only Do Not Disturb really can solve that problem.

That issue can be renamed to `How to disable rocket cat messages about new release` and it will be much correct and really related only to rocket chat, but not to smartphone notification system.

**Comment by gstlouisgit** [comment]
I think I might have found a solution I read about was to disable the cat user.  However I still got a message the other day, but I feel that its less frequent?  I don't know...

**Comment by jorueda** [comment]
Hello there... any news about this topic? Is it posible to disable all the automatic rocket.cat (the bot) messages sent to the users?

**Comment by moerkey** [comment]
I would also appreciate a fix for this because I like to quote:

> There is ALWAYS a new release.

Which is kinda annoying. Therefore, I suggest implementing sth like LTS releases.

Would be nice to get a solution after 4 years. :)
